src/ontology/pato-edit.obo

```diff
@@ -20612,6 +20613,54 @@ synonym: "skin redness" EXACT []
 is_a: PATO:0000051 ! morphology
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: PATO:0040050
+name: diandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have two stamens." [http://orcid.org/0000-0002-2908-3327] {comment="https://orcid.org/0000-0001-8815-0078"}
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040051
+name: monandrous
+def: "A phenotypic sex quality inhering in a  plant organism or a population of plants in which the flowers have only one stamen." [http://orcid.org/0000-0002-2908-3327, https://orcid.org/0000-0001-8815-0078]
+is_a: PATO:0001894 ! phenotypic sex
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: PATO:0040052
+name: protogynous
+def: "A sequential hermaphroditic quality whereby organisms start out as female and change to male at a later life stage." [https://en.wikipedia.org/wiki/Sequential_hermaphroditism] {comment="http://orcid.org/0000-0002-2908-3327"}
+synonym: "protogyny" EXACT []
+is_a: PATO:0070000 ! sequential hermaphrodite
+created_by: https://orcid.org/0000-0001-8815-0078
+
+[Term]
+id: PATO:0040053
+name: protandrous
+def: "A sequential hermaphroditic quality whereby organisms start out as male and change to female at a later life stage." [http://orcid.org/0000-0002-2908-3327, https://en.wikipedia.org/wiki/Sequential_hermaphroditism]
+synonym: "protandry" EXACT []
+is_a: PATO:0070000 ! sequential hermaphrodite
+created_by: https://orcid.org/0000-0001-8815-0078
+
+[Term]
+id: PATO:0040054
```

Comment:
in the OWL file, this ID is PATO_0070036

and the parent is 'neurogliaform morphology' (which seems wrong)